Packs/FiltersAndTransformers/Scripts/AnyMatch/AnyMatch.yml

```
@@ -9,18 +9,16 @@ tags:
   - filter
   - general
 comment: |-
-  Returns all elements from the left side that have a substring that is equal to an element from the right side. Note: This filter is case-insensitive.
-  E.g -AnyMatch left=baby right=A will return baby. For more examples see the filter's Readme.
+  Returns True if one of the elements from the left side has a substring that is equal to an element from the right side. Note: This filter is case-insensitive.
+  E.g -AnyMatch left=baby right=A will return True, because A is a substring of the word baby. For more examples see the filter's Readme.
```

```suggestion
  For example: AnyMatch left=baby right=A will return True, because A is a substring of the word baby. For more examples see the filter's Readme.
```
Where is the filter' s Readme? Do you mean the table of examples in the Script's readme?